processor/groupbytraceprocessor/config.go

```diff
@@ -28,6 +28,10 @@ type Config struct {
 	// Default: 1_000_000.
 	NumTraces int `mapstructure:"num_traces"`
 
+	// NumWorkers is a number of workers processing event queue. Should be equal to physical processors number.
+	// Default: 1.
```

Comment:
This is arguable and might be better to set NumProcs. However, this approach doesn't work with cgroups, this runtime function always tells number of cores on the server

Reply:
And just not to forget - this should be mentioned in the doc